src/SFA.DAS.AssessorService.Web/Controllers/CertificateVersionController.cs

```diff
@@ -66,12 +66,15 @@ private async Task<IActionResult> LoadViewModel(string view)
                 // Only 1 version no need for a selection
                 var singularStandard = certSession.Versions.First();
                 var options = await _standardVersionClient.GetStandardOptions(singularStandard.StandardUId);
-                if (options != null & options.HasOptions())
+                if (options != null && options?.HasOptions() != null)
```

Comment:
Could be done more simply as if (options?.HasOptions() ?? false) and then the second if - should not be required